bokehjs/src/lib/models/annotations/color_bar.ts

```diff
@@ -635,6 +657,42 @@ export class ColorBar extends Annotation {
   override properties: ColorBar.Props
   override __view_type__: ColorBarView
 
+  public axis_mixins(withprefix: boolean = true): {[key: string]: any} {
+    return {
+      ...mixins.attrs_of(this, "major_label_", mixins.Text, withprefix),
+      ...mixins.attrs_of(this, "major_tick_", mixins.Line, withprefix),
+      ...mixins.attrs_of(this, "minor_tick_", mixins.Line, withprefix),
+    }
+  }
+
+  public title_mixins(withprefix: boolean = false): {[key: string]: any} {
+    return mixins.attrs_of(this, "title_", mixins.Text, withprefix)
+  }
+
+  public get axis_properties_refs(): any[] {
+    const properties = this.properties as any
+    const _axis_properties = []
+    const keys = ["major_tick_in", "major_tick_out", "minor_tick_in", "minor_tick_out", "label_standoff",
+                  "major_label_overrides", "major_label_policy", ...Object.keys(this.axis_mixins())]
+
+    for (const key of keys) {
+      if (properties.hasOwnProperty(key)) _axis_properties.push(properties[key])
+    }
+    return _axis_properties
+  }
+
+  public get title_properties_refs(): any[] {
```

Comment:
All those `any` types need to be eradicated. `public` is the default visibility, so no need to repeat this.